src/mockdata/mocks.ts

```diff
@@ -0,0 +1,88 @@
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+import { addDays } from 'date-fns';
+import { WEATHER_MET_API } from '../queries/metApi';
+import { METADATA } from '../queries/metadata';
+import faker from 'faker';
+
+const visualisationTypes = ['barchart', 'thresholdchart', 'piechart', 'linechart'];
+
+const randomListOfWords = () => {
+    const wordList = faker.lorem.words(faker.random.number(7)).split(' ');
+    return wordList.length !== 1 ? wordList : [];
```

Comment:
Isn't there any other way to do this? Now it isn't possible to fake one tag. Probably not important, it's up to you. Another way is to just check if ```wordList[0] === ""```

Reply:
True. Couldn’t find any way to do it with just faker so rushed a quick solution. Didn’t think about the case when only one tag was added. wordList[0] === "" would work.